### PR TITLE
Refactor/install procedure

### PR DESCRIPTION
**Pull request by AlexMartazan**
Changed the installation procedure

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 networks:
   bibbox-default-network:
     external: true
 services:
+  
+  bibbox-sys-commander-angularenv:   # Angular for Frontend building
+    container_name: bibbox-sys-commander-angularenv
+    networks:
+      - bibbox-default-network
+    build:
```
Comment (jungwire) [inline docker-compose.yml:11]: one todo would be if we could manage that no build is needed for the start up so that it is enough that the imaged are pulled from docker hub... but lets discuss and brainstorm about that another time

File: docker-compose_frontend_builder.yml
Comment (jungwire) [inline docker-compose_frontend_builder.yml:1]: Please add a line to the README... e.g. sth like "you can manually trigger the build of the front end with docker compose -f ...."

File: frontend/Dockerfile
Comment (jungwire) [inline frontend/Dockerfile:1]: image
idk any idea what that symbol at the end means?

File: frontend/build.sh
Comment (jungwire) [inline frontend/build.sh:1]: same as above

File: frontend/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Switch to the frontend folder
+cd frontend
+
+# Build the frontend application
+ng build --configuration=production
+
+tail -f /dev/null
```
Comment (jungwire) [inline frontend/build.sh:9]: Is the tail needed? I thought that it just builds the frontend and then the container is done. Is there a need to keep it running endlessly?

File: frontend/health_check.sh
Comment (jungwire) [inline frontend/health_check.sh:1]: Probably answers question above? I am not that familiar with health checks... do you mind talking about (explaining) it to me later?

File: frontend_builder.yml
Comment (jungwire) [inline frontend_builder.yml:1]: identical to docker-compose_frontend_builder.yml.... Please delete one of them

**Review by jungwire — CHANGES_REQUESTED** [review]

File: frontend_builder.yml
Comment (AlexMartazan) [reply frontend_builder.yml:1]: done

File: docker-compose_frontend_builder.yml
Comment (AlexMartazan) [reply docker-compose_frontend_builder.yml:1]: done

File: docker-compose.yml (hunk 1 above)
Comment (AlexMartazan) [reply docker-compose.yml:11]: well, the configuration and what the container needs to do is custom, so unless we upload the already built custom image to the docker hub, i don't see how we're going to do that.

File: frontend/Dockerfile
Comment (AlexMartazan) [reply frontend/Dockerfile:1]: image

It's related to your viewer, maybe end of file or something.

File: frontend/build.sh
Comment (AlexMartazan) [reply frontend/build.sh:1]: same as above ^^

File: frontend/build.sh (hunk 2 above)
Comment (AlexMartazan) [reply frontend/build.sh:9]: Fixed

**Review by jungwire — APPROVED** [review]
Just need to test if it work on an empty machine other wise okay
